package.json

```
@@ -0,0 +1,115 @@
+{
+  "name": "network-dashboard",
+  "description": "Aragon Network Dashboard",
+  "version": "0.1.0",
+  "private": true,
+  "license": "AGPL-3.0-or-later",
+  "author": "Aragon Association <legal@aragon.org>",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/aragon/network-dashboard.git"
+  },
+  "bugs": {
+    "url": "https://github.com/aragon/network-dashboard/issues"
+  },
+  "homepage": "https://github.com/aragon/network-dashboard#readme",
+  "keywords": [
+    "aragon",
+    "network",
+    "ethereum",
+    "dapp",
+    "blockchain",
+    "startups"
+  ],
+  "engines": {
+    "node": ">= 10.0.0"
+  },
+  "bundlewatch": {
+    "files": [
+      {
+        "path": "public/*.js",
+        "maxSize": "1600kb",
+        "compression": "gzip"
+      }
+    ]
+  },
+  "dependencies": {
+    "@aragon/ui": "^1.5.0",
+    "@sentry/browser": "^5.17.0",
+    "core-js": "^3.3.2",
+    "dotenv": "^8.2.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-spring": "^7.2.10",
+    "regenerator-runtime": "^0.13.3",
+    "styled-components": "^4.1.3",
+    "use-wallet": "^0.7.1",
+    "web3": "^1.2.6"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.11.0",
+    "@babel/preset-env": "^7.7.4",
+    "@babel/preset-react": "^7.7.4",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^25.1.0",
+    "babel-plugin-styled-components": "^1.10.6",
+    "bundlewatch": "^0.2.6",
+    "cross-env": "^5.2.0",
+    "eslint": "^5.6.0",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-config-standard-react": "^7.0.2",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jest": "^22.19.0",
+    "eslint-plugin-node": "^7.0.1",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-react": "^7.5.1",
+    "eslint-plugin-react-hooks": "^1.6.0",
+    "eslint-plugin-standard": "^4.0.0",
+    "husky": "^1.0.1",
+    "jest": "^25.1.0",
+    "lint-staged": "^8.1.1",
+    "parcel-bundler": "^1.12.4",
+    "parcel-plugin-bundle-visualiser": "^1.2.0",
+    "prettier": "^1.19.1",
```

I think we could update prettier to 2.x, and remove [the `trailingComma` option](https://github.com/aragon/network-dashboard/pull/1/files?short_path=0e8fec9#diff-f2e285fb35cc9694564cb6d2af099ad2R4).

```suggestion
    "prettier": "^2.0.5",
```